### PR TITLE
Add Easter Egg in About Page

### PR DESCRIPTION
**Pull request by HugoMigner**
### ⁉️ Related Issue
N/A

## 📖 Description
This PR adds a hidden Capra logo in the "About" page, that is triggered upon executing a series of swipes on the App|ETS logo.
To toggle the Easter Egg: 

- Swipe Up
- Swipe Right
- Swipe Down
- Swipe Left
- Tap
On the logo!

_**CONTEXT**_
This Easter Egg was added as an offer to Capra, as they added a hidden App|ETS sticker on their robot. So I added a hidden Capra logo in ETSMobile!

### 🧪 How Has This Been Tested?
N/A

### ☑️ Checklist before requesting a review
- [ x ] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.
- [ ] Do we need to implement analytics?
- [ x ] Make sure to add either one of the following labels: `version: Major`,`version: Minor` or `version: Patch`. 
- [ ] Make sure golden files changes were reviewed and approved.

### 🖼️ Screenshots (if useful):
image

### CODE REVIEW

File: lib/ui/views/about_view.dart [hunk 1]
```diff
@@ -148,6 +166,18 @@ class _AboutViewState extends State<AboutView> with TickerProviderStateMixin {
                   ],
                 ),
               ),
+              if (_easterEggTrigger)
+                SizedBox(
+                  width: 200,
+                  height: 200,
+                  child: Hero(
+                    tag: 'coco',
```
Comment (JonathanDuvalV) [inline lib/ui/views/about_view.dart:174]: peut etre capra au lieu de coco comme tag ?

Comment (HugoMigner) [reply lib/ui/views/about_view.dart]: Oops fixed 😂

**Review by JonathanDuvalV — APPROVED** [review]

**Review by MysticFragilist — APPROVED** [review]
LGTM! :capricorn: :egg: :100:

**Review by JonathanDuvalV — APPROVED** [review]
LGTM :coco_happy:

**Comment by Anthony-karabian** [comment]
LGTM

**Pull request merged**